eth_crowdfund_be/eth_crowdfund_api/src/views/CampaignView.py

```diff
@@ -0,0 +1,74 @@
+from flask import request, json, Response, Blueprint, g, jsonify
+from ..models.Campaign import Campaign, CampaignSchema
+
+
+
+campaign_api = Blueprint('campaigns_api', __name__)
+campaign_schema = CampaignSchema()
+
+@campaign_api.route('/', methods=['POST'])
+def create():
+  req_data = request.get_json()
+  data = campaign_schema.load(req_data)
+  # data, error = campaign_schema.load(req_data)
+  # BUT WITH ERROR: the above code threw an error, currently ignoring errors
+  error = None
+  if error: 
+    return custom_response(error, 400)
+
+  campaign = Campaign(data)
+  campaign.save() 
+  campaign_data = campaign_schema.dump(campaign)
+  return custom_response(campaign_data, 201)
+  
+
+def custom_response(res, status_code):
+  return Response(
+    mimetype="application/json",
+    response=json.dumps(res),
+    status=status_code
+  )
+
+# NOT YET FUNCTIONAL
+@campaign_api.route('/<int:campaign_id>/', methods=['GET'])
```

Comment:
These are formatted very similar to the tutorial. there are errors that need to be fixed. Reference `create` and `all campaigns` for syntax reference